docker-cloud/cloud-swarm/index.md

```diff
@@ -8,15 +8,12 @@ title: Swarms in Docker Cloud (Beta)
 notoc: true
 ---
 
-Docker Cloud now allows you to connect to clusters
-of Docker Engines running in v1.13
-[swarm mode](/engine/swarm/).
-
-With Beta Swarm Mode in Docker Cloud, you can
-provision swarms to popular cloud providers, or
-register existing swarms to Docker Cloud.
-Use your Docker ID to authenticate and securely access
-personal or team swarms.
+Docker Cloud now allows you to connect to clusters of Docker Engines running on
+Docker 17.06 [swarm mode](/engine/swarm/).
```

Comment:
I think I've improved this overall per all of your comments, and got us out of the need to specify all these versions. Let me know what you guys think.